packages/cmr-client/searchConcept.js

```diff
@@ -26,8 +26,8 @@ async function searchConcept({
   headers = {},
   format = 'json',
   recursive = true,
-  cmrLimit,
-  cmrPageSize
+  cmrLimit = process.env.CMR_LIMIT,
```

Comment:
Is there unit test for testing environment variables?

Reply:
added.